tensorrt_llm/_torch/attention_backend/sparse/__init__.py

```
@@ -0,0 +1,11 @@
+from .utils import (get_flashinfer_sparse_attn_attention_backend,
+                    get_sparse_attn_kv_cache_manager,
+                    get_trtllm_sparse_attn_attention_backend,
+                    get_vanilla_sparse_attn_attention_backend)
+
+__all__ = [
+    "get_sparse_attn_kv_cache_manager",
+    "get_vanilla_sparse_attn_attention_backend",
+    "get_trtllm_sparse_attn_attention_backend",
+    "get_flashinfer_sparse_attn_attention_backend",
+]
```

Ruff (0.13.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)